src/dmd/libelf.d

```diff
@@ -11,6 +11,9 @@
 module dmd.libelf;
 
 // Online documentation: https://dlang.org/phobos/dmd_libelf.html
+version(Windows) {}
+else version(OSX) {}
+else:
```

Comment:
Isn't there version identifiers for binary formats?

Reply:
As far as I can see there are `TARGET_*` for the target platform and `OBJ_*` for the object format. But the latter is set at runtime.